src/control/lib/netdetect/netdetect.go

```diff
@@ -1303,6 +1306,8 @@ func DevClassName(class uint32) string {
 		return "EUI64"
 	case Infiniband:
 		return "INFINIBAND"
+	case NetDevAny:
+		return "best-available"
```

Comment:
NetDevAny seems like it really belongs further up the stack, maybe in lib/control, seeing that it's not actually used in the netdetect lib outside of this DevClassName function. As it is, this string seems very tightly coupled to DMG.

Reply:
initially , I did have it in the control API but it felt strange there, I can move it back but I'm not sure it's any better